packages/@aws-cdk/aws-lambda-event-sources/lib/kafka.ts

```diff
@@ -0,0 +1,83 @@
+import * as iam from '@aws-cdk/aws-iam';
+import * as lambda from '@aws-cdk/aws-lambda';
+import * as secretsmanager from '@aws-cdk/aws-secretsmanager';
+import { StreamEventSource, StreamEventSourceProps } from './stream';
+
+export interface KafkaEventSourceProps extends StreamEventSourceProps {
+}
+
+/**
+ * Use a MSK cluster as a streaming source for AWS Lambda
+ */
+export class ManagedKafkaEventSource extends StreamEventSource {
+  /**
+   * Create an event source for MSK
+   * @param clusterArn - the ARN of the MSK cluster
+   * @param topic - the Kafka topic to subscribe to
+   * @param secret - the secret with the Kafka credentials, see https://docs.aws.amazon.com/msk/latest/developerguide/msk-password.html for details
+   * @param props
+   */
+  constructor(
+    readonly clusterArn: string,
+    readonly topic: string,
+    readonly secret: secretsmanager.ISecret,
+    props: KafkaEventSourceProps) {
+    super(props);
+  }
+
+  public bind(target: lambda.IFunction) {
+    target.addEventSourceMapping(
+      `KafkaEventSource:${this.topic}`,
```

Comment:
Include the cluster arn in the id.

if not, this will collide when two clusters with the same topic name are used as event sources for the same lambda function.